sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,158 @@ def __repr__(self):
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+    .. math::
+        F = (\mu_k f_n + (\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2}) \text{sign}(v) + \sigma  v
+
+    where :math:`\mu_k` is the coefficient of kinetic friction, :math:`\mu_s` is the
+    coefficient of static friction, :math:`f_n` is the normal force, :math:`v` is the
+    relative velocity, :math:`v_s` is the Stribeck friction coefficient, and
+    :math:`\sigma` is the viscous friction constant.
+
+    The default friction force is :math:`F = \mu_k f_n`.
+    When specified, the actuator includes:
+
+    - Stribeck effect: :math:`(\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2}`
+    - Viscous effect: :math:`\sigma v`
+
+    Notes
+    =====
+
+    The actuator makes the following assumptions:
+
+    - The actuator assumes relative motion is non-zero.
+    - The normal force is assumed to be a non-negative scalar.
+    - The resultant friction force is opposite to the velocity direction.
+    - Each point in the pathway is fixed within separate objects that are sliding relative to each other. In other words, these two points are fixed in the mutually sliding objects.
+
+    This actuator has been tested for straightforward motions, like a block sliding
+    on a surface.
+
+    The friction force is defined to always oppose the direction of relative velocity :math:`v`.
+    Specifically:
+
+    - The default Coulomb friction force :math:`\mu_k f_n \text{sign}(v)` is opposite to :math:`v`.
+    - The Stribeck effect :math:`(\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2} \text{sign}(v)` is also opposite to :math:`v`.
+    - The viscous friction term :math:`\sigma v` is opposite to :math:`v`.
+
+    Examples
+    ========
+
+    The below example shows how to generate the loads produced by a Coulomb kinetic
+    friction actuator in a mass-spring system with friction.
+
+    >>> import sympy as sm
+    >>> from sympy.physics.mechanics import (dynamicsymbols, ReferenceFrame, Point,
+    ...     LinearPathway, CoulombKineticFriction, LinearSpring, KanesMethod, Particle)
+
+    >>> x, v = dynamicsymbols('x, v', real=True)
+    >>> m, g, k, mu_k, mu_s, v_s, sigma = sm.symbols('m, g, k, mu_k, mu_s, v_s, sigma')
+
+    >>> N = ReferenceFrame('N')
+    >>> O, P = Point('O'), Point('P')
+    >>> O.set_vel(N, 0)
+    >>> P.set_pos(O, x*N.x)
+    >>> P.set_vel(N, v*N.x)
+
+    >>> pathway = LinearPathway(O, P)
+    >>> friction = CoulombKineticFriction(mu_k, m*g, pathway, v_s=v_s, sigma=sigma, mu_s=mu_k)
+    >>> spring = LinearSpring(k, pathway)
+    >>> block = Particle('block', point=P, mass=m)
+
+    >>> kane = KanesMethod(N, (x,), (v,), kd_eqs=(x.diff() - v,))
+    >>> friction.to_loads()
+        [(O, (g*m*mu_k*sign(sign(x(t))*Derivative(x(t), t)) + sigma*sign(x(t))*Derivative(x(t), t))*x(t)/Abs(x(t))*N.x), (P, (-g*m*mu_k*sign(sign(x(t))*Derivative(x(t), t)) - sigma*sign(x(t))*Derivative(x(t), t))*x(t)/Abs(x(t))*N.x)]
```

Comment:
This doctest, which tests the solution when no assumptions about +/- are made on the variables should be a unit test also. Copy this test to the unit tests to test the most general case. After that, I'm good with merging.